step-9-selenium-tests.md

```diff
@@ -61,7 +61,7 @@ public class SystemTest {
             .withNetworkAliases("kafka");
 
     @ClassRule
-    public static BrowserWebDriverContainer browser = (BrowserWebDriverContainer) new BrowserWebDriverContainer()
+    private static BrowserWebDriverContainer browser = new BrowserWebDriverContainer<>()
```

Comment:
Doesn't `@ClassRule` need the public modifier?

Reply:
Crikey, you're right. #16 will replace, but I'll change anyway.